tensornetwork/network_operations.py

```diff
@@ -91,6 +92,84 @@ def transpose(node: BaseNode,
       backend=node.backend.name)
   return new_node.reorder_axes(perm)
 
+def copy(nodes: Iterable[BaseNode], conjugate: bool = False) -> Tuple[dict, dict]:
+  """
+
+  Return a copy of the TensorNetwork.
+  Args:
+    nodes: An `Iterable` (Usually a `List` or `Set`) of `Nodes`.
+    conjugate: Boolean. Whether to conjugate all of the nodes in the
+      `TensorNetwork` (useful for calculating norms and reduced density
+      matrices).
+  Returns:
+    A tuple containing:
+      node_dict: A dictionary mapping the nodes of the original
+                 network to the nodes of the copy.
+      edge_dict: A dictionary mapping the edges of the original
+                 network to the edges of the copy.
+  """
+  #TODO: add support for copying CopyTensor
+  if conjugate:
+    node_dict = {
+        node: Node(
+            node.backend.conj(node.tensor),
+            name=node.name,
+            axis_names=node.axis_names,
+            backend=node.backend.name) for node in nodes
+    }
+  else:
+    node_dict = {
+        node: Node(
+            node.tensor, name=node.name, axis_names=node.axis_names, 
+            backend=node.backend.name)
+        for node in nodes
+    }
+  edge_dict = {}
+  for edge in get_all_edges(nodes):
+    node1 = edge.node1
+    axis1 = edge.node1.get_axis_number(edge.axis1)
+
+    if not edge.is_dangling():
+      node2 = edge.node2
+      axis2 = edge.node2.get_axis_number(edge.axis2)
+      new_edge = Edge(node_dict[node1], axis1, edge.name,
+                      node_dict[node2], axis2)
+      new_edge.set_signature(edge.signature)
+    else:
+      new_edge = Edge(node_dict[node1], axis1, edge.name)
+
+    node_dict[node1].add_edge(new_edge, axis1)
+    if not edge.is_dangling():
+      node_dict[node2].add_edge(new_edge, axis2)
+    edge_dict[edge] = new_edge
+  return node_dict, edge_dict
+
+def remove_node(node: BaseNode
+               ) -> Tuple[Dict[Text, Edge], Dict[int, Edge]]:
+  """Remove a node from the network.
+
+  Args:
+    node: The node to be removed.
+
+  Returns:
+    broken_edges_by_name: A Dictionary mapping `node`'s axis names to
```

Comment:
Same comment about "broken." The connotation with "broken" makes it sound like something has gone wrong.

Reply:
Done.